hazelcast/src/test/java/com/hazelcast/config/ConfigCompatibilityChecker.java

```diff
@@ -182,6 +182,12 @@ private static boolean isCompatible(HotRestartConfig c1, HotRestartConfig c2) {
         return c1 == c2 || (c1Disabled && c2Disabled) || (c1 != null && c2 != null && nullSafeEqual(c1.isFsync(), c2.isFsync()));
     }
 
+    private static boolean isCompatible(MergePolicyConfig c1, MergePolicyConfig c2) {
```

Comment:
Where is this private method used?

Reply:
It will be used by all implementations with an additional line like:
```java
                    && ConfigCompatibilityChecker.isCompatible(c1.getMergePolicyConfig(), c2.getMergePolicyConfig());
```
I had to pull it out here, so all dependent PRs have it available.

Reply:
:+1: